saleor/core/taxes/vatlayer/interface.py

```diff
@@ -132,16 +133,20 @@ def apply_taxes_to_shipping_price_range(
 def assign_tax_to_object_meta(obj: Union["Product", "ProductType"], tax_code: str):
     if tax_code not in dict(TaxRateType.CHOICES):
         return
-    if not hasattr(obj, "meta"):
+    if not hasattr(obj, "private_meta"):
```

Comment:
No need to put taxes in private_meta. Tax object just stores a category type of product which can be visible

Reply:
👍